sdk/src/test/java/com/finbourne/lusid/tutorials/ibor/Valuation.java

```diff
@@ -83,6 +87,86 @@ public void run_aggregation_with_buy() throws ApiException
                 });
     }
 
+    @Test
+    public void run_aggregation_on_multicurrency_portfolio_with_stock_in() throws ApiException {
+        //  setup test environment scope
+        String uuid = UUID.randomUUID().toString();
+        String quotesScope = uuid;
+
+        // begin test data preparation...
+
+        //  prepare portfolio in baseline GBP currency
+        String portfolioId = String.format("Id-%s", uuid);
+        String portfolioDisplayName = String.format("Portfolio-%s", uuid);
+        CreateTransactionPortfolioRequest transactionPortfolioRequest = createTransactionPortfolioRequest(portfolioId, portfolioDisplayName, "GBP");
+
+        //  prepare transactions  across GBP and USD. transactions are cash funding and stock fundings
```

Comment:
these are transactions are both transfer in of both cash and stock, perhaps rephrase the comment as:

"prepare transactions across GBP and USD, transactions are transfer in of cash and stock"

Reply:
Done